package.json

```diff
@@ -216,5 +217,15 @@
       "cloudteam-td-agent-setup",
       "dial-agent"
     ]
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "git add"
```

Comment:
Not sure that we want to auto-add the newly linted files. This should be down to the developer.

Reply:
Done